docs/sdk/analytics/fragments/ios/getting-started.md

```diff
@@ -12,8 +12,8 @@ Amazon Pinpoint is a fully managed AWS service that you can use to engage with y
 
 The Amplify CLI helps setup and configure Pinpoint within your application and connect with the AWS Mobile SDK.
 
-<b>Prerequisite:</b> [Install and configure the Amplify CLI](..)<br>
-<b>Recommendation:</b> [Complete the Getting Started guide](./start)
+ - **Prerequisite:** [Install and configure the Amplify CLI](/cli/start/install)
+ - **Recommendation:** [Complete the Getting Started guide](/start?integration=ios)
```

Comment:
We do not need to guide them to the getting started anymore. It's clearly visible in the top nav.